Index: MODELES/LMDZ.GENERIC/libf/phystd/physiq_mod.F90
===================================================================
--- MODELES/LMDZ.GENERIC/libf/phystd/physiq_mod.F90	(révision 2484)
+++ MODELES/LMDZ.GENERIC/libf/phystd/physiq_mod.F90	(copie de travail)
@@ -2175,7 +2175,7 @@
       if(callrad.and.(.not.newtonian))then
       
          !call writediagfi(ngrid,"ALB","Surface albedo"," ",2,albedo_equivalent)
-         !call writediagfi(ngrid,"ALB_1st","First Band Surface albedo"," ",2,albedo(:,1))
+         call writediagfi(ngrid,"ALB","Surface albedo"," ",2,albedo(:,1))
          call writediagfi(ngrid,"ISR","incoming stellar rad.","W m-2",2,fluxtop_dn)
          call writediagfi(ngrid,"ASR","absorbed stellar rad.","W m-2",2,fluxabs_sw)
          call writediagfi(ngrid,"OLR","outgoing longwave rad.","W m-2",2,fluxtop_lw)
@@ -2183,8 +2183,8 @@
          
 !           call writediagfi(ngrid,"ASRcs","absorbed stellar rad (cs).","W m-2",2,fluxabs_sw1)
 !           call writediagfi(ngrid,"OLRcs","outgoing longwave rad (cs).","W m-2",2,fluxtop_lw1)
-!           call writediagfi(ngrid,"fluxsurfsw","sw surface flux.","W m-2",2,fluxsurf_sw)
-!           call writediagfi(ngrid,"fluxsurflw","lw back radiation.","W m-2",2,fluxsurf_lw)
+         call writediagfi(ngrid,"SSR","sw surface rad.","W m-2",2,fluxsurf_sw)
+         call writediagfi(ngrid,"SLR","lw surface rad.","W m-2",2,fluxsurf_lw)
 !           call writediagfi(ngrid,"fluxsurfswcs","sw surface flux (cs).","W m-2",2,fluxsurf_sw1)
 !           call writediagfi(ngrid,"fluxsurflwcs","lw back radiation (cs).","W m-2",2,fluxsurf_lw1)
 
